tests/security/cc/netfilter.pm

```diff
@@ -0,0 +1,147 @@
+    } else {
+        mutex_wait('NETFILTER_SERVER_READY');
+
+        # export the variables
+        my $client_first  = $data->{client}->{first_interface};
+        my $client_second = $data->{client}->{second_interface};
+        my $server_first  = $data->{server}->{first_interface};
+        my $server_second = $data->{server}->{second_interface};
+
+        # Deal with ipv4 address: change 192.168.0.1/24 to 192.168.0.1
+        $client_first->{ipv4}  =~ s/\/.*//g;
+        $client_second->{ipv4} =~ s/\/.*//g;
+        $server_first->{ipv4}  =~ s/\/.*//g;
+        $server_second->{ipv4} =~ s/\/.*//g;
+
+        script_run("export PASSWD=$testapi::password");
+        script_run("export LOCAL_DEV=$client_first->{netcard}\@eth0");
+        script_run("export LOCAL_SEC_DEV=$client_second->{netcard}\@eth0");
+        script_run("export LOCAL_SEC_MAC=$client_second->{mac_add}");
+        script_run("export LOCAL_IPV4=$client_first->{ipv4}");
+        script_run("export LOCAL_IPV6=$client_first->{ipv6}");
+        script_run("export LOCAL_SEC_IPV4=$client_second->{ipv4}");
+        script_run("export LOCAL_SEC_IPV6=$client_second->{ipv6}");
+
+        script_run("export LBLNET_SVR_IPV4=$server_first->{ipv4}");
+        script_run("export LBLNET_SVR_IPV6=$server_first->{ipv6}");
+        script_run("export SECNET_SVR_IPV4=$server_second->{ipv4}");
+        script_run("export SECNET_SVR_IPV6=$server_second->{ipv6}");
+        script_run("export SECNET_SVR_MAC=$server_second->{mac_add}");
+
+        script_run('export BRIDGE_FILTER=toebr');
+
+        run_audit_test($lblnet_pid, $cmd);
+
+        # Compare current test results with baseline
+        $result = compare_run_log('netfilter');
+    }
+
+    $self->result($result);
+}
+
+#
+# This function is only used by client.
+# When one case finishes, the port maybe hasn't been released, so we need to restart
+# `lblnet_tst_server` in case the following test case ends with ERROR.
+# So the test cases in netfilter will be run one by one.
+#
+sub run_audit_test {
+    my ($pid, $lblnet_cmd) = @_;
+    my $output = script_output('./run.bash --list');
+    my @lines  = split(/\n/, $output);
+    my @test_lists;
+    foreach (@lines) {
+        my $num;
+        if ($_ =~ /\[(\d+)\]\s+\S+/) {
+            $num = $1;
+        } else {
+            next;
+        }
+        my $cmd    = "./run.bash $num";
+        my $result = script_output($cmd, timeout => 600);
+
+        # If the port is busy, the test case will end with error, we need to restart lblnet_tst_server
+        if (index($result, 'could not setup remote test server') != -1) {
+            script_run("kill -9 $pid");
+            $pid = background_script_run($lblnet_cmd);
+            # give more seconds for killing this process
+            sleep 3;
+            assert_script_run($cmd, timeout => 600);
+        }
+    }
+
+    # the test case 8 sometimes will end with ERROR because of the performance issue. we need to run it again
```

Comment:
Use capital letter for the first charactor.

 Btw, `s/the test case 8/ The 8th test case/` and also add the test case name here for avoiding confusing.